index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#welcome-section">Contact</a
+          >
+        </li>
+        <li>
+          <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#projects"
+            >Work</a
+          >
+        </li>
+        <li>
+          <a href="https://cdpn.io/freeCodeCamp/fullpage/zNBOYG#contact"
+            >About</a
+          >
+        </li>
```

Comment:
[REQUIRED] Nice job with the navigation bar. I think it would be better to properly append the semantic tags under their first parent element. This would make the code cleaner and readable. Why don't you do that by removing the extra white spaces.

Reply:
- [ ] Good job 💪🏿